labels/queries/balancer_pools.sql

```diff
@@ -0,0 +1,52 @@
+-- TODO: improve for cases where there are multiple binds/rebinds/unbinds in the same block
+with events as (
+    -- binds
+    select call_block_number as block_number, 
+    contract_address as pool, token, denorm
+    from balancer."BPool_call_bind"
+    where call_success
+
+    union all
+
+    -- rebinds
+    select call_block_number as block_number, 
+    contract_address as pool, token, denorm
+    from balancer."BPool_call_rebind"
+    where call_success
+
+    union all
+    
+    -- unbinds
+    select call_block_number as block_number, 
+    contract_address as pool, token, 0 as denorm
+    from balancer."BPool_call_unbind"
+    where call_success
+),
+state_with_gaps as (
+    select events.block_number, events.pool, events.token, events.denorm,
+    LEAD(cast(events.block_number as text), '1', '99999999') over (partition by events.pool, events.token order by events.block_number) as next_block_number
+    from events 
+), 
+settings as (
+    select pool, 
+    coalesce(t.symbol,'???') as symbol, 
```

Comment:
Would it be better to inner join (i.e. only have labels with both symbols), or do you think ???/YFI is fine for now?

Reply:
I think it's fine for now. It helps distinguish between 2/3/4... token pools.